honeybee/radiance/datatype.py

```diff
@@ -4,20 +4,57 @@
 import warnings
 import os
 
-__all__ = ['RadDescrPath','RadDescrNum','RadDescrBoolFlags','RadDescrNumTuple']
+__all__ = ['RadiancePath', 'RadianceNumber', 'RadianceBoolFlag',
+           'RadianceNumericTuple']
 
-class RadDescrDefault(object):
+
+class RadianceDefault(object):
     """
-    The default descriptor which defines the __get__ method. Since the get
-    method is not likely to be fancy most descriptors can inherit this class.
+    The default descriptor for radiance commands. Provides base case attributes
+    for other descriptors.
+
+    Attributes:
+        _name: Required for all cases. Name of the flag, like 'ab' for '-ab 5'
```

Comment:
@mostaphaRoudsari The _ is only meant for private use when coding the descriptors as other wise the script goes into maximum recursion depth by trying to run setattr over and over. See the `checkExists` [here](https://github.com/sariths/honeybee-core/blob/2d53c01ef1864936d0856fbddb73b91d09f4f3ca/honeybee/radiance/command/rcollate.py#L24-L25) and [here](https://github.com/sariths/honeybee-core/blob/2d53c01ef1864936d0856fbddb73b91d09f4f3ca/honeybee/radiance/datatype.py#L264). There are a few workarounds for not using the _ but that involves invoking the super class each time ( which I found to be more confusing while coding).